java/org/apache/catalina/filters/CsrfPreventionFilter.java

```diff
@@ -87,11 +104,170 @@ public void setNonceRequestParameterName(String parameterName) {
         this.nonceRequestParameterName = parameterName;
     }
 
+    /**
+     * Sets the flag to enforce CSRF protection or just log failures as DEBUG
+     * messages.
+     *
+     * @param enforce <code>true</code> to enforce CSRF protections or
+     *                <code>false</code> to log DEBUG messages and allow
+     *                all requests.
+     */
+    public void setEnforce(boolean enforce) {
+        this.enforce = enforce;
+    }
+
+    /**
+     * Gets the flag to enforce CSRF protection or just log failures as DEBUG
+     * messages.
+     *
+     * @return <code>true</code> if CSRF protections will be enforced or
+     *         <code>false</code> if all requests will be allowed and
+     *         failures will be logged as DEBUG messages.
+     */
+    public boolean getEnforce() {
```

Comment:
Is is common throughout Tomcat source code istead of `isEnforce()` for boolean return value?

Reply:
There are 308 classes in the Tomcat 11.0.x source tree which contain `boolean isFoo()` and only 141 which contain `boolean getFoo()`. I didn't bother checking is any contain both.